client/src/pages/Profile.js

```diff
@@ -158,6 +178,13 @@ const Profile = () => {
           return <div key={attribute} />;
         })
       )}
+      {userPermissions.edit.length > 0 ? (
+        <Button id="submit-button" type="large" onClick={submitChanges}>
```

Comment:
Not a necessity, but I think it would be really helpful from UI standpoint to enable/disable this button based on whether any fields have changed. You could store the user object every time it's updated as state, and then compare the current user object to it using isEqual from lodash (https://lodash.com/docs/4.17.15#isEqual) to determine if the button is disabled or not.